README.md

```diff
@@ -164,9 +164,9 @@ const items = [
 
 render(
   <Downshift
-    onChange={selection => alert(
-      selection ? `You selected ${selection.value}` : 'Selection Cleared'
-    )}
+    onChange={selection =>
```

Comment:
not needed change

Reply:
I think it was made by `prettier`. I can temporary remove `lint-staged` if you want.